projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx

```diff
@@ -11,8 +12,7 @@ import { View, Text } from 'react-native';
 /**
  * Internal dependencies
  */
-import { VideoPressIcon as icon } from './components/icons';
-import { VIDEOPRESS_VIDEO_ALLOWED_MEDIA_TYPES } from './constants';
+import VideoPressUploader from './components/videopress-uploader/index.native';
```

Comment:
Non-blocking but curious about why not use `./components/videopress-uploader` for the import path. Shouldn't it resolve to the native version?

Reply:
I could use `./components/videopress-uploader` as it imports the native version, but TypeScript was complaining in the editor due to missing props. In this case, since the component has different props for each platform, TypeScript by default infers the prop types from the default version of the file (i.e. the web version).

<img width="1345" alt="Screenshot 2023-03-15 at 09 46 15" src="https://user-images.githubusercontent.com/14905380/225256029-7abb53f5-d88e-4c39-b6a5-b992c264ea16.png">

Reply:
Yep, that's it. In my case using VSCode, it's there where I see the error. Using any of both imports doesn't seem to affect the logic.